Doc/library/pathlib.rst

```diff
@@ -1477,6 +1477,21 @@ example because the path doesn't exist).
       available. In previous versions, :exc:`NotImplementedError` was raised.
 
 
+.. method:: Path.copy(target, *, follow_symlinks=True)
+
+   Copy the contents of this file to the *target* file. If *target* specifies
+   a file that already exists, it will be replaced.
+
+   If *follow_symlinks* is false, and this file is a symbolic link, *target*
+   will be created as a symbolic link. If *follow_symlinks* is true and this
+   file is a symbolic link, *target* will be a copy of the symlink target.
+
+   This method doesn't guarantee whether permissions, ownership, and other
+   metadata is copied; it may vary by operating system and Python version.
```

Comment:
Can we specify our intent (I'm thinking it'll be something along the lines of "using an efficient method provided by the OS, so if the OS thinks permissions/etc. are important enough to copy by default, then this function will do it"), or alternatively, simply say that we will follow OS defaults and the exact behaviour should be tested on the OS in question.

I want to ensure we have enough detail here to help guide future changes to the behaviour, and also to protect ourselves against having to recall a release because an OS doesn't do things "properly" through their own APIs.